web/src/components/asset/AssetUploadAction.tsx

```
@@ -0,0 +1,145 @@
+        "handleFileReject: files list non-empty but first file is falsy",
+      );
+      return;
+    }
+
+    const accepted = acceptedMIMEs.reduce((prev: string[], curr: string) => {
+      const extensions = mime[curr]?.extensions;
+      if (!extensions) {
+        return prev;
+      }
+
+      return [...prev, ...extensions];
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)